README.md

```diff
@@ -1,20 +1,48 @@
 # Ember RFCs
 
+The RFC ("request for comments") process is how Ember designs and
+achieves consensus on "substantial" proposed changes.
+
 Many changes, including bug fixes and documentation improvements can be
 implemented and reviewed via the normal GitHub pull request workflow.
 
 Some changes though are "substantial", and we ask that these be put
 through a bit of a design process and produce a consensus among the Ember
 core teams.
 
-The "RFC" (request for comments) process is intended to provide a
-consistent and controlled path for new features to enter the framework.
+The process is intended to provide a consistent and controlled path for new
+features and changes to enter the framework. RFCs can be opened by any member
+of the community.
 
 [Active RFC List](https://github.com/emberjs/rfcs/pulls)
 
 [Approved RFC List](https://emberjs.github.io/rfcs/)
```

Comment:
These links will need to be changed since PRs are no longer the best way to check things. Also merged PRs are not necessarily approved after Staging is implemented. We should probably point to rfcs.emberjs.com.

Reply:
I addressed this in the latest commit. lmk if you think that is enough.

Reply:
Looks great.